pods-template.txt

```diff
@@ -132,8 +132,8 @@ Please edit when you make changes to any pod templates
 	[if live_status_date]
 		<div class="odhBoxSingleCard liveStatus">
 	[else]
-		[if intake_status_date]
```

Comment:
This can't be the only change to the template; this is only a change to the auto-update field, but the actual pod definition itself needs to change if this is changing